arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -296,5 +296,36 @@ public async Task<GetGraphEdgeCollectionsResponse> GetGraphEdgeCollectionsAsync(
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Updates the data of the specific vertex in the collection.
+        /// PATCH/_api/gharial/{graph}/vertex/{collection}/{vertex}
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="graphName"></param>
+        /// <param name="collectionName"></param>
+        /// <param name="vertexKey"></param>
+        /// <param name="body"></param>
+        /// <param name="query"></param>
+        /// <returns></returns>
+        public async Task<PatchVertexResponse<T>> PatchVertexAsync<T>(string graphName, string collectionName, string vertexKey, T body, PatchVertexQuery query = null)
+        {
+            string uri = _graphApiPath + '/' + WebUtility.UrlEncode(graphName) +
+                "/vertex/" + WebUtility.UrlEncode(collectionName) + "/" + WebUtility.UrlEncode(vertexKey);
+            if (query != null)
+            {
+                uri += "?" + query.ToQueryString();
+            }
+            StringContent content = GetStringContent(body, true, true);
```

Comment:
I don't think we should ignore null or camel-case convert for user-defined objects